cmd/docker_options.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+      "fmt"
+
+      "github.com/creasty/defaults"
+      "github.com/falcosecurity/driverkit/validate"
+      "github.com/go-playground/validator/v10"
+      logger "github.com/sirupsen/logrus"
+)
+
+var dockerOptions *DockerOptions
+
+// DockerOptions represent the configuration flags for the driverkit docker subcommand.
+type DockerOptions struct {
```

Comment:
Probably the docker options are already available in the go source code of docker.

Could we use that so to inherit the exact same behavior?

Reply:
Some places where I'd look for them are:

- https://github.com/docker/cli/tree/master/opts
- https://github.com/moby/moby/tree/master/api/types/container

A quick search brought me this:

https://github.com/docker/cli/blob/88c6089300a82d3373892adf6845a4fed1a4ba8d/cli/command/container/opts.go#L631